package.json

```
@@ -25,6 +25,7 @@
     "svg-path-bounding-box": "1.0.4",
     "svglint": "1.0.6",
     "svgo": "1.3.2",
+    "svgpath": "^2.3.0",
```

Can you pin this new (direct) dependency to `2.3.0`?

```suggestion
    "svgpath": "2.3.0",
```

---

Sure :)